### PR TITLE
Sign in page

### PR DESCRIPTION
**Pull request by AviadCohen24**
Created sign in page

### CODE REVIEW

File: Frontend/beard-requests/src/App.js [hunk 1]
```diff
@@ -1,10 +1,25 @@
...
+import Drawer from './Components/drawer/Drawer.component';
 
 function App() {
-  return (
-  <div>
 
-  </div>
+  const [isDrawerOpen, setIsDrawerOpen] = useState(false);
+
+  const handleDrawerClick = () => setIsDrawerOpen(!isDrawerOpen);
```
Comment (AlexShavkuta) [inline Frontend/beard-requests/src/App.js:12]: Use prevStare instead

File: Frontend/beard-requests/src/Components/form-input/FormInput.component.jsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import { InputContainer, Input, Label } from "./FormInput.styles";
+
+const FormInput = ({description, type, placeholder, onChange}) => {
+
+    const handleInputChange = (event) => onChange(event.target.value)
```
Comment (AlexShavkuta) [inline Frontend/beard-requests/src/Components/form-input/FormInput.component.jsx:5]: When there is a single parameter don’t use parentheses

File: Frontend/beard-requests/src/Routes/nav-bar/NavBar.component.jsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import { BarContainer, BeardLogo, LogoContainer, BeardProjectName, HamburgerButton, HamburgerIcon, SideBarContainer } from './NavBar.styles.js'
+
+import { useState } from 'react'
+
+const NavBar = ({ changeOpenHandler, drawerState }) => {
+
+    const handleDrawerClick = () => changeOpenHandler();
```
Comment (AlexShavkuta) [inline Frontend/beard-requests/src/Routes/nav-bar/NavBar.component.jsx:7]: Why did you implement it that way?

File: Frontend/beard-requests/src/Routes/nav-bar/NavBar.styles.js
Comment (AlexShavkuta) [inline Frontend/beard-requests/src/Routes/nav-bar/NavBar.styles.js:1]: Excellent use of styled components!

File: Frontend/beard-requests/src/Utils/Server.js [hunk 4]
```diff
@@ -0,0 +1,8 @@
+import axios from "axios";
+
+const serverUrl = "";
+
+export const checkAuthentication = async(personalNumber, password) => {
```
Comment (AlexShavkuta) [inline Frontend/beard-requests/src/Utils/Server.js:5]: Add validation for the parameters, so you won’t be able to send to the server empty parameters.

**Review by AlexShavkuta — CHANGES_REQUESTED** [review]

File: Frontend/beard-requests/src/Utils/Server.js (hunk 4 above)
Comment (ShacharDahan) [reply Frontend/beard-requests/src/Utils/Server.js:5]: in line 14 of Form.component.jsx there is validation checks for the parameters

**Pull request closed without merging**